src/views/room.vue

```diff
@@ -56,18 +38,18 @@
 </template>
 
 <script>
-import BaseButton from '../components/baseButton.vue'
+import AdminControls from '../components/adminControls.vue'
 import EstimationChart from '../components/estimationChart.vue'
-import EstimationNumbers from '../components/estimationNumbers.vue'
+import EstimationValues from '../components/estimationValues.vue'
```

Comment:
the component itself is still named `EstimationNumbers`  which is not that big of a deal but should be changed to keep things nice.
See line 21 in `src/components/estimationValues.vue`

Reply:
fixed!